packages/apps/immutable-records/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,36 @@
+import { backdrop, container } from './Modal.css';
+
+import type { FC, PropsWithChildren } from 'react';
+import { useEventListener } from 'usehooks-ts';
+
+interface ModalProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+export const Modal: FC<PropsWithChildren<ModalProps>> = ({
+  children,
+  open,
+  onClose,
+}) => {
+  useEventListener('keydown', (e) => {
```

Comment:
Do we really need another dependency to use `addEventListener`?

Reply:
`usehooks-ts` provides many useful utilities for react apps at a 4kb gzipeed size. I don't think there should be much reason to have to justify a package of that size if it makes us have to write and maintain less code.